content/source/docs/enterprise/saml/identity-provider-configuration-aad.html.md

```diff
@@ -7,53 +7,109 @@ page_title: "SAML Azure Active Directory Identity Provider Configuration - Terra
 
 Follow these steps to configure Azure Active Directory (AAD) as the identity provider (IdP) for Terraform Enterprise.
```

Comment:
Dropping this here as an FYI and suggestion. Microsoft already has this great tutorial about configuring SSO with TFE! https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/terraform-enterprise-tutorial

I wonder if instead of documenting the process again, we could just link to that tutorial from this page and then add any necessary supplementary content for TFE users here? That way, we wouldn't be duplicating existing content. Thoughts? @im2nguyen

Reply:
Could we also do a statement in the description that this will close this issue: https://github.com/hashicorp/terraform-website/issues/1942

Reply:
I like the idea of linking out to Microsoft, that way, they can own and manage these set of docs since they're more about working with AAD than TFE itself